canonical-web-and-design/jaas-dashboard
# Update z-index of modal

## Done

Update z-index of modal

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Verify side nav no longer covers the modal

## Details

Fixes #732
